src/query/storages/common/table-meta/src/caches/meta_cache.rs

```diff
@@ -30,7 +30,7 @@ pub type TableSnapshotCache = InMemoryItemCacheHolder<TableSnapshot>;
 pub type TableSnapshotStatisticCache = InMemoryItemCacheHolder<TableSnapshotStatistics>;
 /// In memory data cache of bloom index data.
 /// For each indexed data block, the index data of column is cached individually
-pub type BloomIndexCache = InMemoryBytesCacheHolder;
+pub type BloomIndexCache = InMemoryItemCacheHolder<Xor8>;
```

Comment:
`Xor8` is one implementation of `Filter`. Can we use the `Filter` instead of using the Xor8 directly?

Reply:
thanks for reminding me.   A new crate `cache-manager` is being extracted, after that, the BloomIndexCachewill hold Xor8Filter directly (it depends on crate `storages-common-index`)